app/models/experimentation/notifications_experiment.rb

```diff
@@ -81,17 +84,25 @@ def monitor
       end
     end
 
+    def messages_report_key
+      if start_time > NEW_MESSAGE_REPORTING_DATE
+        NEW_MESSAGE_REPORTING_DATE
+      else
+        OLD_MESSAGE_REPORTING_KEY
+      end
+    end
+
     def record_notification_results
       # TODO: Look at query performance
```

Comment:
Is this still pending?

Reply:
Haven't looked at it closely but it hasn't caused any problems even with 50k messages/day. It's decently quick to finish. Will remove it.